src/sklearn_evaluation/plot/clustering.py

```diff
@@ -78,28 +87,47 @@ def elbow_curve(X, clf, n_clusters=None, n_jobs=1, show_cluster_time=True, ax=No
     ax: matplotlib Axes
         Axes containing the plot
 
+    Notes
+    -----
+    .. deprecated:: 0.8.7
+        'n_clusters' renamed to 'range_n_clusters' and will be removed in version 0.8.9
+
     Examples
     --------
     .. plot:: ../examples/elbow_curve.py
 
     """
-    if n_clusters is None:
-        n_clusters = range(1, 10, 2)
+
+    if n_clusters is not None:
+        if range_n_clusters is not None:
+            raise AttributeError(
```

Comment:
deprecations should show warnings, not throw exceptions, follow  the guidelines from here https://ploomber-core.readthedocs.io/en/latest/deprecation.html#parameter-renamed

Reply:
@yafimvo fix those 2 in a new PR

Reply:
@edublancas in the [original issue](https://github.com/ploomber/sklearn-evaluation/issues/148) you wrote to throw an error when users pass both `n_clusters` and `range_n_clusters`, telling them to only use `range_n_clusters`.

should I remove this error and throw a warning instead?

Reply:
sorry for the back and forth. I had forgotten I said we'd raise an exception. I discussed this with Ido yesterday. the idea is to use this [guide](https://ploomber-core.readthedocs.io/en/latest/deprecation.html#parameter-renamed) for all deprecations - it contains functions to encapsulate the process. I'm gonna make some changes so let's put this on hold for a bit

Reply:
There's also a parameter that should be renamed from `labels` to `label`